website/content/docs/configuration/storage/raft.mdx

```diff
@@ -116,6 +116,10 @@ delay) mode. The maximum allowed value is 10.
   breaking a large value into chunks. By default, the chunk size is the same as
   Raft's max size log entry. The default value for this configuration is 1048576
   -- two times the chunking size.
+  - **Note:** This option corresponds to [Consul's `kv_max_value_size` parameter](https://developer.hashicorp.com/consul/docs/agent/config/config-files#kv_max_value_size) for
+    Vault clusters using a Consul storage backend.  If you are migrating from Consul
+    storage to Raft Integrated Storage, ensure this value or its default aligns with
+    your Consul server configuration.
```

Comment:
```suggestion
    Vault clusters using a Consul storage backend.  If you are migrating from Consul
    storage to Raft Integrated Storage,  and have changed this value in Consul from its
    default to a value larger than the Integrated Storage default of 1MB, then you will 
    need to make the same change in Vault's Integrated Storage config.
```

Reply:
I've made a small change here to hopefully make it clearer that customers don't need to change the default unless they have special requirements